.github/workflows/wheels.yml

```diff
@@ -0,0 +1,113 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }} ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 99
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            arch: x86_64
+          - os: ubuntu-latest
+            arch: i686
+          - os: ubuntu-latest
+            arch: aarch64
+          - os: ubuntu-latest
+            arch: ppc64le
+          - os: ubuntu-latest
+            arch: s390x
+
+          - os: macOS-latest
+            arch: x86_64
+          - os: macOS-latest
+            arch: arm64
+          - os: macOS-latest
+            arch: universal2
+
+          - os: windows-2019
+            arch: AMD64
+          - os: windows-2019
+            arch: x86
+          # Does not build currently
+          # - os: windows-2019
+          #   arch: ARM64
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v2
+
+      # Used to host cibuildwheel
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==2.12.0
+
+      - name: Build wheels
+        run: python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_ARCHS: ${{ matrix.arch }}
+          # TODO: Disable building PyPy wheels. If the build system gets modernized, this should be
+          # auto-detected based on the Cython dependency.
+          CIBW_SKIP: pp*
+          CIBW_TEST_REQUIRES: pytest
+          CIBW_TEST_COMMAND: pytest {project}
+
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./wheelhouse/*.whl
+
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Build sdist
+        run: pipx run build --sdist
```

Comment:
I'm less familiar with pipx, but basically we should add 
https://github.com/capnproto/pycapnp/blob/master/.github/workflows/packagingtest.yml#L71
 (`--force-bundled-libcapnp`) when building pycapnp.

The correct behaviour is that since no local version of libcapnp is installed the build scripts should pull a bundled version. We want a bundled version as we have no idea which version that's installed (or not); which is far more preferable when installing packages through pypi.
I think some of this info was lost over the past few refactors of the GitHub Actions.

Reply:
Just to be clear: This particular action only build the source distribution, not any binary wheel. It's purpose is for people that have a machine that is incompatible with any of the pre-compiled wheels.

I assume that you want to add `--force-bundled-libcapnp` in order to distribute the sources files of the upstream Capn'Proto release, so that they don't need to be downloaded separately? In principle, that sounds okay (but not critical), but I don't believe this is something that has ever happened in the past, and I don't believe that adding `--force-bundled-libcapnp` will achieve this. This flag will download and compile Capt'nProto, but the source code will still not be copied into the source distribution (I just tested this). As such, in it's current form this flag is useless for the source distribution.

Note that when building the binary wheel distribution, the flag is relevant. But I believe it still doesn't need to be added, because none of the CI images used here include Capt'nProto, and hence the build system will automatically build from source. Even if that wasn't the case, `cibuildwheel` would automatically repair the wheel to include the system-level Capt'nProto binaries into the wheel (that's the magic if this, it all happens automatically).